### ISSUE TITLE
Groups page diagram zooms instead of scrolling

### ISSUE THREAD

**Issue by dtuite** [backstage/backstage#11429]
## Expected Behavior

On the groups page, when I scroll up with the mouse hovered over the diagram, the page should scroll up.

## Actual Behavior

The diagram zooms in.

## Steps to Reproduce

Here's a recording of me demonstrating the bug.

https://www.loom.com/share/aa4744982fa04611bbbaa26736c3b584

There is some demo data here which might be useful when reproducing this issue: https://github.com/RoadieHQ/demo-data/blob/main/examples/roadie.yaml

## Context

I get stuck on the Groups page for a few seconds every time I try to exit 😄 

## Your Environment

- Browser Information: 

- Output of `yarn backstage-cli info`: 

```text
Google Chrome Version 101.0.4951.54 (Official Build) (arm64)
Mac
```

![Screenshot 2022-05-10 at 17 31 43](https://user-images.githubusercontent.com/562403/167678090-ba893a80-0bb0-43d2-a2f7-bec3b4bf7d8e.png)

**Comment by Rugvip** [comment]
Thank you for the video, that does help! :grin:

Really torn on whether this is fine and expected behavior for a fullscreen diagram, or something to fix. To be more precise, we should probably either fix the zooming, or the scrolling. I'm leaning towards making sure that the page doesn't scroll being the best solution here?

**Comment by dtuite** [comment]
@Rugvip Yeah that might be a good solution. If it worked like Google maps it might be ok.

**Comment by ki504178** [comment]
Hi!!
May I work on this issue?

**Comment by tan31989** [comment]
Hi folks, I was trying to check if I could work on this. But, when I look at this, I thought the behaviour of zoom on a diagram could be a valid use case (personal thought). Any ideal workflow tools/graph, ideally allows us to scroll using mouse scroll. Not sure if I am missing something specific with this use case.

**Comment by awanlin** [comment]
https://user-images.githubusercontent.com/67169551/181010478-8a15f874-c9b7-4e01-834d-1d7926b5e8b8.mov

Thoughts I'd add a quick example of this. I know the zoom seems unreasonable it was just the easiest way to demo the issue. But as you can see you can only scroll if you know about the magic space on the left or right side. Where you run into this is when you have a more detailed Org structure we have 100+ groups so when you get to this page you zoom around a lot but easily get stuck :(

I just used the demo site for this video for reference: https://demo.backstage.io/explore/groups

**Comment by tan31989** [comment]
Hmm, @Rugvip, would we expect the zoom to not take place OR have a clear border that says this entire place can be zoomed?

**Comment by freben** [comment]
@tan31989 What he meant is, if this is on a fullscreen view as in the video above, it should probably be forced to adapt the view to the available viewport and not even being _possible_ to scroll it - and in those circumstances, zooming becomes natural

**Comment by vorbrodt** [comment]
Hello, new contributor here! :)

I would like to be assigned to this issue if it’s deemed suitable for a new contributor with experience working in frontend. 

Just so I understand the issue correctly. The diagram should match the viewport and through that make it not possible to scroll up and down on the page, but instead zoom in and out on the diagram only. Or is there something more to it that I need to keep in mind?

**Comment by Rugvip** [comment]
@vorbrodt Thank you! I think it's a good one, there might be a bit of fiddling with styles, but hopefully no deep knowledge of the Backstage codebase required. And yep, that's my understanding too.

**Comment by Rugvip** [comment]
Fixed in #14403
